content/kubernetes/security/ldap.md

```diff
@@ -0,0 +1,166 @@
+
+### Bind credentials
+
+For LDAP servers that require authentication for client queries, store the bind credentials in a secret and reference them in the `RedisEnterpriseCluster` custom resource.
+
+1. Create a secret to store the bind credentials.
+    
+    ```sh
+    kubectl -n <my-rec-namespace> create secret generic ldap-bind-credentials \
+        --from-literal=dn='<disinguished-name>' \
+        --from-literal=password=<password>
+    ```
+    The secret must:
+    - Reside within the same namespace as the `RedisEnterpriseCluster` custom resource.
+    - Include a `dn` key with the distinguished name for the user performing the query (such as `cn=admin,dc=example,dc=org`).
+    - Include a `password` key with the bind password.
+
+    Replace the `<placeholders>` in the command above with your own values.
+
+1. Reference the secret name in the `.spec.ldap.bindCredentialsSecretName` field of the `RedisEnterpriseCluster` custom resource.
+
+    ```yaml
+    spec:
+      ldap:
+        bindCredentialsSecretName: ldap-bind-credentials
+    ```
+
+### LDAPS or STARTTLS protocols
+
+In addition to plain LDAP protocol, Redis Enterprise Software also supports LDAPS and STARTTLS protocols for secure communication with the LDAP server.
+
+To enable one of these protocols, edit the `spec.protocol` field in the `RedisEnterpriseCluster` custom resource:
+
+#### Enable `LDAPS`
+
+  ```yaml
+      spec:
+        protocol: LDAPS
+  ```
+
+  Default port: 636
+
+#### Enable `STARTTLS`
+
+  ```yaml
+      spec:
+        protocol: STARTTLS
+  ```
+
+  Default port: 389
+
+### CA certificate
+
+To use a custom CA certificate for validating the LDAP server certificate, store the CA certificate in a secret and reference the secret in the `RedisEnterpriseCluster` custom resource.
+
+1. Create a secret to hold the CA certificate.
+
+    ```sh
+    kubectl -n <my-rec-namespace> create secret generic ldap-ca-certificate \
+        --from-file=cert=<ca-cert>.pem
+    ```
+
+    The secret must:
+    - Reside within the same namespace as the `RedisEnterpriseCluster` custom resource.
+    - Include a `cert` key with a PEM-encoded CA certificate (such as `cacert.pem`).
+
+    Replace the `<placeholders>` in the command above with your own values.
+
+1. Reference the secret name in the `spec.ldap.caCertificateSecretName` field of the `RedisEnterpriseCluster` custom resource.
+
+    ```yaml
+    spec:
+      ldap:
+        caCertificateSecretName: ldap-ca-certificate
+    ```
+
+### Client certificates
+
+To use an LDAP client certificate, store the certificate in a secret and reference the secret in the `RedisEnterpriseCluster` custom resource.
+
+1. Create a secret to hold the client certificate.
+
+    ```sh
+    kubectl -n <my-rec-namespace> create secret generic ldap-client-certificate \
+      --from-literal=name=<ldap_client \
+      --from-file=certificate=<client-cert>.pem \
+      --from-file=key=<private-key>.pem
```

Comment:
There's no necessity that the files name will have a .pem suffix, although typically they would. Nevertheless, consider leaving the .pem suffix to the examples below.

```suggestion
      --from-file=certificate=<client-cert-file> \
      --from-file=key=<private-key-file>
```